atlas/AtlasMl/tests/test_artemis_client.py

```
@@ -0,0 +1,125 @@
+            assert len(result) == 2
+            assert result[0].title == "Exercise 1"
+            assert result[1].title == "Exercise 2"
+
+    @pytest.mark.asyncio
+    async def test_health_check_success(self, artemis_client):
+        """Test successful health check."""
+        mock_response = MagicMock()
+        mock_response.status_code = 200
+
+        with patch('httpx.Client') as mock_client:
+            mock_client.return_value.__aenter__.return_value.get.return_value = mock_response
```

Ruff (0.12.2)

Line too long (93 > 88)

(E501)